HLTrigger/Configuration/python/CustomConfigs.py

```diff
@@ -110,21 +110,13 @@ def HLTDropPrevious(process):
 
 
 def MassReplaceInputTag(process,old="rawDataCollector",new="rawDataRepacker",verbose=False,moduleLabelOnly=False,skipLabelTest=False):
-#   replace InputTag values (adapted from Configuration/Applications/python/ConfigBuilder.py)
-    from PhysicsTools.PatAlgos.tools.helpers import massSearchReplaceAnyInputTag
-    for s in process.paths_().keys():
-        massSearchReplaceAnyInputTag(getattr(process,s),old,new,verbose,moduleLabelOnly,skipLabelTest)
-    for s in process.endpaths_().keys():
-        massSearchReplaceAnyInputTag(getattr(process,s),old,new,verbose,moduleLabelOnly,skipLabelTest)
+    from PhysicsTools.PatAlgos.tools.helpers import massReplaceInputTag
+    massReplaceInputTag(process, old, new, verbose, moduleLabelOnly, skipLabelTest)
     return(process)
 
 def MassReplaceParameter(process,name="label",old="rawDataCollector",new="rawDataRepacker",verbose=False):
-#   replace values of named parameters
-    from PhysicsTools.PatAlgos.tools.helpers import massSearchReplaceParam
-    for s in process.paths_().keys():
-        massSearchReplaceParam(getattr(process,s),name,old,new,verbose)
-    for s in process.endpaths_().keys():
-        massSearchReplaceParam(getattr(process,s),name,old,new,verbose)
+    from PhysicsTools.PatAlgos.tools.helpers import massReplaceParameter
+    massReplaceParameter(process, name, old, new, verbose)
```

Comment:
What is the reason for this change?
For some reason we looped explicitly over both paths and endpaths!